core/src/apps/benfen/tx_parser.py

```
@@ -0,0 +1,432 @@
+        expiration = parse_transaction_expiration(parser)
+
+        return {
+            "version": version,
+            "kind_type": hex(kind_type),
+            "sender": sender,
+            "inputs": inputs,
+            "commands": commands,
+            "gas_data": gas_data,
+            "expiration": expiration,
+        }
+    except Exception:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)